src/views/ManageList.jsx

```diff
@@ -1,7 +1,74 @@
+import { addItem } from '../api/firebase';
+import toast, { Toaster } from 'react-hot-toast';
+
 export function ManageList() {
+	const notify = () => toast.success('Item added to list!');
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		const itemInfoForm = new FormData(e.target);
+
+		console.log({
+			itemName: itemInfoForm.get('item'),
+			daysUntilNextPurchaseSelect: itemInfoForm.get('when-to-buy-select'),
+			daysUntilNextPurchaseRadio: itemInfoForm.get('when-to-buy-radio'),
+		});
+		// addItem({ itemName: itemInfoForm.get("item"), daysUntilNextPurchase: itemInfoForm.get("when-to-buy")});
+		notify();
+	};
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<div>
+			<p>
+				Hello from the <code>/manage-list</code> page!
+			</p>
+			<form onSubmit={handleSubmit}>
+				<label>
+					Item: <input id="item" type="text" name="item" required />
+				</label>
+				<br />
+
+				<label htmlFor="when-to-buy">
```

Comment:
I added this option because I know we were trying to decide which we prefered and I wanted to see it. I do like the select drop down but they are both passing their values in the formData of the submit.